### PR TITLE
adjust checkout page

### PR DESCRIPTION
**Pull request by ryan-m-burns**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -162,26 +159,12 @@ <h2>Payment Details</h2>
 
       <!-- Checkout Summary -->
       <section class="order-section">
-        <!-- edit from here -->
         <div class="checkout-cart-container">
           <div class="checkout-cart-header" id="order-summary-right">
             <h1>Order Summary</h1>
             <h4>This is a Pickup Order</h4>
             <div class="checkout-order-summary">
-              <div class="order-items">
```
Comment (ryan-m-burns) [inline public/index.html:171]: Cleaned up some hard coded data

File: db/seeds/05_01_cart_items.sql [hunk 2]
```diff
@@ -1,30 +1,30 @@
 INSERT INTO cart_items (cart_id, quantity, menu_item_id) VALUES (
   1,
-  100,
+  10,
```
Comment (ryan-m-burns) [inline db/seeds/05_01_cart_items.sql:3]: more reasonable quantities for testing purposes

File: public/index.html [hunk 3]
```diff
@@ -133,55 +130,204 @@ <h3 class="restaurant-name"> Succulent Society Cafe </h3>
...
+        </div>
+        <div class="payment-details">
+          <h2>Payment Details</h2>
+        </div>
+      </section>
       <!-- Checkout Summary -->
       <section class="order-section">
-        <!-- edit from here -->
         <div class="checkout-cart-container">
-          <div class="checkout-cart-header" id="order-summary-right">
```
Comment (ryan-m-burns) [inline public/index.html:167]: wasn't sure what this was here for so cleaned it up for now until we review together.

**Comment by onaroll2021** [comment]
thank you. we can discuss these codes in the code.

**Review by onaroll2021 — APPROVED** [review]

**Pull request merged**